listener.py

```diff
@@ -35,16 +33,13 @@ async def run():
 
     killer = GracefulKiller()
 
-    inspect_db_session = get_inspect_session()
-    trace_db_session = get_trace_session()
-
-    inspector = MEVInspector(rpc, inspect_db_session, trace_db_session)
+    inspector = MEVInspector(rpc, None, None)
     base_provider = get_base_provider(rpc)
 
     while not killer.kill_now:
         await inspect_next_block(
             inspector,
-            inspect_db_session,
+            None,
```

Comment:
need this for `find_latest_block_update` and `update_latest_block`

Reply:
Agh, good catch! Forgot to fix this one -- will take a stab at it